.github/action_scripts/send_transactions.js

```diff
@@ -0,0 +1,375 @@
+        await sleep( SLEEP_TIME_UNTIL_QUERY );
+
+        const originBalance = await metagraphTokenClient.getBalance();
+        const destinationBalance = await metagraphTokenClient.getBalanceFor( destination.address );
+
+        return { originBalance, destinationBalance };
+    } catch( error ) {
+        const errorMessage = `Error when sending transactions between wallets, message: ${error}`;
+        logMessage( errorMessage );
+        throw error;
+    }
+};
+
+const sendDoubleSpendTransaction = async ( networkOptions ) => {
+    const lastRef = await dag4.network.getAddressLastAcceptedTransactionRef( FIRST_WALLET_ADDRESS );
+
+    const accountFirstNode = dag4.createAccount();
+    accountFirstNode.loginSeedPhrase( FIRST_WALLET_SEED_PHRASE );
+    await accountFirstNode.connect(
+        {
+            networkVersion: '2.0',
+            l0Url: networkOptions.l0GlobalUrl,
+            l1Url: networkOptions.dagL1UrlFirstNode,
+            testnet: true
+        }
+    );
+    const signedTransaction = await accountFirstNode.generateSignedTransaction(
+        SECOND_WALLET_ADDRESS,
+        5000,
+        1,
+        lastRef
+    );
+
+    const accountSecondNode = dag4.createAccount();
+    accountSecondNode.loginSeedPhrase( FIRST_WALLET_SEED_PHRASE );
+    await accountSecondNode.connect(
+        {
+            networkVersion: '2.0',
+            l0Url: networkOptions.l0GlobalUrl,
+            l1Url: networkOptions.dagL1UrlSecondNode,
+            testnet: true
+        }
+    );
+    const signedTransaction2 = await accountSecondNode.generateSignedTransaction(
+        THIRD_WALLET_ADDRESS,
+        5000,
+        1,
+        lastRef
+    );
+
+    try {
+        await Promise.all( [
+            dag4.network.postTransaction( signedTransaction ),
+            dag4.network.postTransaction( signedTransaction2 )
+        ] );
+
+        logMessage( `Waiting ${SLEEP_TIME_UNTIL_QUERY}ms until fetch wallet balances` );
+        await sleep( SLEEP_TIME_UNTIL_QUERY );
+
+        const firstWalletBalance = await accountFirstNode.getBalanceFor( FIRST_WALLET_ADDRESS );
+        const secondWalletBalance = await accountFirstNode.getBalanceFor( SECOND_WALLET_ADDRESS );
+        const thirdWalletBalance = await accountFirstNode.getBalanceFor( THIRD_WALLET_ADDRESS );
+
+        if( secondWalletBalance === 9900 && thirdWalletBalance === 5000 ){
+            logMessage( `Amount sent to third wallet ${thirdWalletBalance} and not to second: ${secondWalletBalance}` );
+            logMessage( `FirstWalletBalance: ${firstWalletBalance}` );
+            logMessage( `SecondWalletBalance: ${secondWalletBalance}` );
+            logMessage( `ThirdWalletBalance: ${thirdWalletBalance}` );
+            return;
+        }
+
+        if( secondWalletBalance === 14900 && thirdWalletBalance === 0 ){
+            logMessage( `Amount sent to second wallet ${thirdWalletBalance} and not to third: ${secondWalletBalance}` );
```

Comment:
`second wallet ${thirdWalletBalance}`
shouldn't it be `secondWalletBalance`? the same with third at the end of log message